src/content/docs/en/guides/on-demand-rendering.mdx

```diff
@@ -38,7 +38,7 @@ All other pages are statically-generated at build time!
 
 The following example shows opting out of prerendering in order to display a random number each time the endpoint is hit:
 
-```js title="src/pages/randomnumber.js" {1}
+```js title="src/pages/randomnumber.js" ins={1}
```

Comment:
nit: The previous example use `ins={2}`, shouldn't we use the same visual information here? (or the other way around... not covered by my suggestion)